util/rollup.config.js

```diff
@@ -0,0 +1,42 @@
+import babel from 'rollup-plugin-babel';
```

Comment:
There seems to be a fair amount of commonality between the rollup configs (`pacakges/d3fc/rollup.config.js`, `pacakges/d3fc/dev.rollup.config.js` & `utili/rollup.config.js`). I can imagine we'll have issues if the config drifts. Is there an easy way we can remove some of the duplication?

Reply:
I think I can provide a config flag here and merge the two files, will give that a shot today